README.md

```diff
@@ -14,18 +15,21 @@ Follow the [installation instructions](https://docs.datadoghq.com/serverless/ins
 
 We're excited to share that our next-generation, low-overhead Lambda Extension is now generally available. This new extension is packaged alongside our existing extension, yet offers substantially faster cold starts as well as lower resource consumption.
 
-You can opt out of the next-generation Lambda Extension by setting the the environment variable `DD_EXTENSION_VERSION: compatibility` and using extension version `v67` or later.
+You can opt out of the next-generation Lambda Extension by setting the environment variable `DD_EXTENSION_VERSION: compatibility` and using extension version `v67` or later.
 
 ### Supported Configurations
+
 Today, all workloads using Logs and Metrics are supported.
 
 APM Tracing is supported for Python, NodeJS, Go, Java, and .NET runtimes.
-Profiling and ASM are not yet supported, and will fallback to compatability mode.
+Profiling and ASM are not yet supported, and will fallback to compatibility mode.
```

Comment:
actually can you remove `Profiling` here? Only ASM fails over now.

Reply:
Will do